### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by muskan2532150**
- Adding the style to the project .

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -19,7 +19,7 @@ input,
 button {
   font-size: 25px;
   border: 7px solid black;
-  padding: 12px;
+  padding: 11px;
 }
```
Comment (codecaiine) [inline src/style.css]: - [ ] Good job so far but your app is not working as expected :cry: Your application displays some Javascript code  **[object HTMLInputElement]:[object Object]** as you can see in the screenshots. Please fix that to make your app work as expected. :+1: 

Screenshot from 2022-08-25 13-06-49

- [ ] I would like to recommend you arrange your app design. Please try to stick to the wireframe as much as possible cause it is very important for the developer to peer attention to details :smile: 
You can check the screenshots below to see the differences :arrow_down: 

#### Wireframe
Screenshot from 2022-08-25 13-17-29

#### Your design
Screenshot from 2022-08-25 13-17-56

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @muskan2532150 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use of gitflow :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Required Changes ♻️

- [ ] The link to your license is broken :cry: Please don't forget to add a license to your project to make it more professional and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/). :smile: 
- [ ] I would like to recommend you add the instruction to run the app locally , that will also make your readme professional and will help non-technical people :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (muskan2532150) [reply src/style.css]: @codecaiine,thank you for the review but for [object HTMLInputElement]:[object Object] I can't delete that as it already post in API and delete action is not available in that given URL

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @muskan2532150 👋, 

Well done for implementing the required changes 👏 
Your project is complete now complete, and there is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**